packages/datatrak-web/src/features/Survey/Components/SurveyQuestionGroup.tsx

```diff
@@ -46,8 +46,8 @@ export const SurveyQuestionGroup = ({ questions }: { questions: SurveyScreenComp
           focus: () => void;
         };
       };
-      if (firstError && firstError?.ref) {
-        firstError?.ref?.focus();
+      if (firstError?.ref) {
+        firstError?.ref.focus();
```

Comment:
```suggestion
        firstError.ref.focus();
```
Don't need to optional chain again right?